### PR TITLE
README for compliance

### PR DESCRIPTION
**Pull request by sistawendy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+* *Main Contact:* Maura Hubbell
+
+* *Scope:* internal
+
+* *Products*: Fresh Web Explorer, Fresh Web Alerts, Moz Mentions,
+OSE Mentions
+
+* *Replaces*: N/A
+
+* *Open Source*: No
```
Comment (dlecocq) [inline README.md]: This is a public repo, so I'd say 'Yes.'

**Comment by dlecocq** [comment]
One nit, otherwise LGTM

File: README.md [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+permutes them appropriately, and stores them or looks up near-
+matches in any of a variety of databases.
+
+* *Also known as:* N/A
+
+* *Team:* Big Data
+
+* *Main Contact:* Maura Hubbell
+
+* *Scope:* internal
```
Comment (b4hand) [inline README.md]: Scope should probably be External as well since this repo is public. ;)

File: README.md (hunk 1 above)
Comment (sistawendy) [inline README.md]: *Smack*. I got it right for `g-crawl-py`, just not this one.

**Pull request merged**